ext/openssl/ossl_pkey.c

```diff
@@ -399,6 +399,10 @@ ossl_pkey_check_public_key(const EVP_PKEY *pkey)
 
     /* OpenSSL < 1.1.0 takes non-const pointer */
     ptr = EVP_PKEY_get0((EVP_PKEY *)pkey);
+    if(ptr == NULL) {
```

Comment:
Pardon this style note: Other `if` statements are following by a space character in this file. I don't know the preferred form, just noted this being slightly different from the other if conditionals.